packages/synthetic-chain/package.json

```diff
@@ -2,20 +2,17 @@
   "name": "@agoric/synthetic-chain",
   "version": "0.0.5-4",
   "description": "Utilities to build a chain and test proposals atop it",
-  "bin": "./cli.ts",
-  "main": "cli.ts",
+  "bin": "./dist/cli/cli.js",
+  "main": "./dist/cli/cli.js",
   "type": "module",
+  "module": "./dist/lib/index.js",
```

Comment:
Let's add an `exports` field to prevent external users doing deep imports into `dist`

Reply:
I was reluctant to make that big a change yet, but I suppose consumers that it would break just won't upgrade.

Reply:
it'll break consumers already with the move, might as well make it official. Introducing `exports` is technically semver major, might as well bite the bullet once